### PR TITLE
Added a Register company functionality

### PR DESCRIPTION
**Pull request by kelkel17**
## Description / Purpose (checkpoints)
 - This PR will allow users to register their company and add a single admin.

Fixes issue:  (issue link)

## Type of change
 - New functionality (breaking change which fixes an issue)

## How To Test This PR

 **Commands**
 - `php artisan migrate:fresh`
 - `npm run dev` / `npm run watch`

 **Routes/Url**
 - `/register/company`

**Notes**
 - The payment type is still static since we did not include a payment method yet.

### CODE REVIEW

File: resources/js/components/Courses.vue [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                        <tbody>
+                            <tr>
+                                <td>1234</td>
+                                <td>Test Course</td>
+                                <td>This is a test Course</td>
+                                <td>100</td>
+                                <td>Sun Asterisk</td>
+                                <td>July 20, 2019</td>
+                            </tr>
+                            <!-- <tr v-for="user, index in users" :key="user.id">
```
Comment (christanplaza) [inline resources/js/components/Courses.vue:41]: Are we going to be using this line of code?

File: resources/js/components/Courses.vue [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                            <button :disabled="form.busy" type="submit" class="btn btn-primary">{{ editMode ? 'Update' : 'Create User' }}</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
```
Comment (christanplaza) [inline resources/js/components/Courses.vue:114]: Are these scripts just placeholders for now?

File: resources/js/components/Courses.vue (hunk 1 above)
Comment (kelkel17) [reply resources/js/components/Courses.vue:41]: I will used it for a reference on the next functionality that I will do which is course management.

File: resources/js/components/Courses.vue (hunk 2 above)
Comment (kelkel17) [reply resources/js/components/Courses.vue:114]: What do you mean by placeholders?

File: resources/js/components/Courses.vue (hunk 1 above)
Comment (christanplaza) [reply resources/js/components/Courses.vue:41]: Noted.

File: resources/js/components/Courses.vue (hunk 2 above)
Comment (christanplaza) [reply resources/js/components/Courses.vue:114]: You have functions and data for Users but aren't using it in the vue instance and I believe it's just copy-pasted from the Users.vue and you still have to refactor the functions.

**Review by christanplaza — APPROVED** [review]
Approved

**Review by RunstyleX — APPROVED** [review]
Approved

**Pull request merged**